bin/package

```diff
@@ -38,18 +38,16 @@ fi
 
 echo "Packaging $PKG_VERSION"
 
+rm -rf dist || true
 mkdir -p dist || exit 1
 
-./bin/build || exit 1
-chmod +x *.phar
+if [ ! -d "build" ]; then
+  exit 1
+fi
 
 echo "Creating tarballs"
 
-# Support BSD tar because OS X :(
-if [[ $(tar --version) = bsdtar* ]]; then
-  tar -s "/.*/psysh/" -czf dist/psysh-${PKG_VERSION}.tar.gz psysh.phar
-  tar -s "/.*/psysh/" -czf dist/psysh-${PKG_VERSION}-compat.tar.gz psysh-compat.phar
-else
-  tar --transform "s/.*/psysh/" -czf dist/psysh-${PKG_VERSION}.tar.gz psysh.phar
-  tar --transform "s/.*/psysh/" -czf dist/psysh-${PKG_VERSION}-compat.tar.gz psysh-compat.phar
-fi
+tar -czf dist/psysh-${PKG_VERSION}.tar.gz build/psysh.phar
```

Comment:
maybe that could be a one liner... not sure about it

Reply:
The reason for the previous code was to rename all the phars on the way in to just be `psysh`. Unfortunately the options to do this aren’t portable, hence the “if BSD” business.